includes/rest-api/class-wp-job-manager-filters-status.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Declaration of our Configuration Filter
+ *
+ * @package WPJM/REST
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class WP_Job_Manager_Models_Settings
+ */
+class WP_Job_Manager_Filters_Status extends WPJM_REST_Model_Declaration {
+
+	/**
+	 * Declare our fields
+	 *
+	 * @param  WPJM_REST_Model_Field_Declaration_Collection_Builder $def Def.
+	 * @return array
+	 * @throws WPJM_REST_Exception Exc.
+	 */
+	function declare_fields( $def ) {
```

Comment:
I'm just going to put this review in once, but it should apply across all the classes here (and everywhere!)

You should always explicitly declare method visibility.Aka, make it explicitly `public` :)